onadata/libs/authentication.py

```diff
@@ -74,3 +91,25 @@ def authenticate(self, request):
                 'to use basic authentication.'
             ))
         return user_auth
+
+    def authenticate_credentials(self, userid, password, request=None):
+        user, _ = super().authenticate_credentials(
```

Comment:
This file does have `import ugettext as _`; did we ever figure out how to reconcile that convention with the Python convention of using `_` as something to discard?

Reply:
Dissatisfying discussion: https://softwareengineering.stackexchange.com/a/147271

Reply:
I brought it up in https://chat.kobotoolbox.org/#narrow/stream/4-KoBo-Dev/topic/Python.20coding.20style/near/80459.

Now that I've actually read the rest of the function, I wouldn't use the `_` variable here since you're not throwing it away—you actually return it along with `user`. I think it's called `auth` by convention, or at least [DRF puts it](https://www.django-rest-framework.org/api-guide/authentication/#how-authentication-is-determined) into `request.auth`.